plasma/child_chain/child_chain.py

```diff
@@ -93,15 +98,27 @@ def submit_block(self, block):
         if not valid_signature:
             raise InvalidBlockSignatureException('failed to submit block')
 
-        self.root_chain.transact({'from': '0x' + self.authority.hex()}).submitBlock(block.merkle.root, self.current_block_number)
+        self.root_chain.transact({'from': '0x' + self.authority.hex()}).submitBlock(block.merkle.root)
         # TODO: iterate through block and validate transactions
         self.blocks[self.current_block_number] = self.current_block
-        self.current_block_number += 1
+        self.current_block_number += self.child_block_interval
         self.current_block = Block()
 
     def get_transaction(self, blknum, txindex):
         return rlp.encode(self.blocks[blknum].transaction_set[txindex]).hex()
 
+    def get_tx_pos(self, transaction):
+        decoded_tx = rlp.decode(utils.decode_hex(transaction), Transaction)
+
+        for blknum in self.blocks:
+            block = self.blocks[blknum]
+            for txindex in range(0, len(block.transaction_set)):
```

Comment:
This is more of an optimization but it might be better to store the `transaction_set` in a dictionary as opposed to an array.

Reply:
A dict keyed by the transaction's hash would work here.